README.md

```diff
@@ -1,2 +1,104 @@
-# website-template
-A template to create all public facing sites
+# Tiny Site Backend Setup Guide
+
+This README provides step-by-step instructions for setting up the Tiny Site Backend on your local development environment. The Tiny Site Backend is built using the Go programming language, PostgreSQL for the database, and Google authentication for authentication. Before you begin, ensure that you have the following prerequisites installed on your system:
+
+1. Go 1.21 or higher
+2. PostgreSQL 15 or higher
+3. pgAdmin (optional but recommended for database management)
+
+## Prerequisites
+
+### Go 1.21
```

Comment:
`Go 1.21` 
This heading change is remaining I think, rest looks good.

Reply:
Not needed